app/controllers/api/v1/locations_controller.rb

```diff
@@ -18,4 +18,9 @@ def create
   def index
     render json: Location.all.order(created_at: :desc)
```

Comment:
For these methods, might be more conventional/readable to keep with the standard controller method definition like so:

```
@locations = Location.all.order(created_at: :desc)
render json: @locations
```